python/ray/tune/schedulers/async_hyperband.py

```diff
@@ -50,6 +54,14 @@ def __init__(self,
         assert grace_period > 0, "grace_period must be positive!"
         assert reduction_factor > 1, "Reduction Factor not valid!"
         assert brackets > 0, "brackets must be positive!"
+        assert mode in ["min", "max"], "mode must be 'min' or 'max'!"
+
+        if reward_attr is not None:
+            mode = "max"
+            metric = reward_attr
+            logger.warning("`reward_attr` will be depreciated!"
```

Comment:
```suggestion
            logger.warning("`reward_attr` is deprecated and will be removed in a future version of Tune. Setting `metric={}` and `mode=max`.".format(reward_attr))
```
(I think formatting might be broken here in this comment)